tests/helpers/images.ts

```diff
@@ -20,3 +21,46 @@ export const deleteAllImages = async (
   const notification = page.locator(".toast-notification");
   await expect(notification).toHaveText(/.*deleted.*/);
 };
+
+export const createImage = async (page: Page, instance: string) => {
+  //Creates an image from an instance
```

Comment:
```suggestion
export const createImageFromInstance = async (page: Page, instance: string) => {
```
Naming it to what it does lets us remove the comment. I think this function should live in the instance helper file, as it is working on the instance detail page.